### PR TITLE
CDAR-736: Create volumes for carla recorder data collection

### PR DESCRIPTION
**Pull request by paulbourelly999**
# PR Details
## Description

Create volumes for Carla to write Carla log and for scenario runner to write criteria json. This will be used for data analysis.
## Motivation and Context
Using carla-recorder and carla-scenario-runner metrics module for data analysis.

## How Has This Been Tested?
CDASim deployment 

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: ail_vru_uc1_scenario/docker-compose.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ services:
     volumes:
       - /tmp/.X11-unix:/tmp/.X11-unix
       - /opt/carma-simulation/logs:/opt/carma-simulation/logs
+      - ./carla-recorder/:/app/scenario_runner/metrics/data/
```
Comment (adamlm) [inline ail_vru_uc1_scenario/docker-compose.yml:16]: Why are we mounting this volume?

**Review by adamlm — CHANGES_REQUESTED** [review]

File: ail_vru_uc1_scenario/docker-compose.yml (hunk 1 above)
Comment (paulbourelly999) [reply ail_vru_uc1_scenario/docker-compose.yml:16]: The scenario runner image creates a criteria json file and the carla server running in the CDA Sim container creates a carla recording. Using the criteria json file we can parse the log file with the metrics module in carla-scenario-runner to return information about metrics.

Comment (paulbourelly999) [reply ail_vru_uc1_scenario/docker-compose.yml:16]: Behind the scenes when I specify --record <path> carla-scenario-runner calls carla-client.start_recording(<SCENARIO_RUNNER_ROOT>/<path>) and stop recording once the scenario completes. This prompts the carla server to create a recording file under this path.

File: ail_vru_uc1_scenario/README.md [hunk 2]
```diff
@@ -18,9 +18,13 @@ This CARMA Configuration Image creates a **AIL** (Anything-In-the-Loop) scenario
 | SUMO      | 1.15       |
 
 ## Deployment Instructions
-
+### Deployment Steps
+TODO: Complete this section
```
Comment (adamlm) [inline ail_vru_uc1_scenario/README.md:22]: We should flesh this out before merging.

**Review by adamlm — CHANGES_REQUESTED** [review]

File: ail_vru_uc1_scenario/README.md (hunk 2 above)
Comment (paulbourelly999) [reply ail_vru_uc1_scenario/README.md:22]: I do not think so. This is more related to the entire deployment than this data collection.

**Review by adamlm — APPROVED** [review]

**Pull request merged**